source/group_eq_bot/driver/local/reader.py

```diff
@@ -0,0 +1,49 @@
+from dataclasses import dataclass
+from typing import Union
+from pathlib import Path
+from io import BufferedReader
+
+from driver.local.audio.base import BaseAudioReader
+from driver.local.text.base import BaseTextReader
+from driver.models.supported_extensions import  SupportedFilesExtensions
+
+
+@dataclass
+class Reader:
+    path: Path  # full path to a particular file to open
+
+    def __post_init__(self) -> None:
+        self.path = Path(self.path)
+
+    def check_file_existance(self) -> None:
```

Comment:
Please separate the function functionalities on 2 dedicated functions.
The function has to handle 1 particular task.

For example:
```python
def file_exists():
    pass

def path_exists():
    pass

```

Reply:
U can keep just file_exists()

Reply:
Function was redundant, so only this remained
```python
def file_exists():
    pass
```